cmd/server/pactasrv/blobs.go

```diff
@@ -22,23 +22,23 @@ func (s *Server) AccessBlobContent(ctx context.Context, request api.AccessBlobCo
 		blobIDs = append(blobIDs, pacta.BlobID(item.BlobId))
 	}
 	err404 := oapierr.NotFound("blob not found", zap.Strings("blob_ids", asStrs(blobIDs)))
-	bos, err := s.DB.BlobContexts(s.DB.NoTxn(ctx), blobIDs)
+	bcs, err := s.DB.BlobContexts(s.DB.NoTxn(ctx), blobIDs)
 	if err != nil {
 		if db.IsNotFound(err) {
 			return nil, err404
 		}
 		return nil, oapierr.Internal("error getting blob owners", zap.Error(err), zap.Strings("blob_ids", asStrs(blobIDs)))
 	}
 	asMap := map[pacta.BlobID]*pacta.BlobContext{}
-	for _, boi := range bos {
```

Comment:
https://www.youtube.com/watch?v=WZpLQa_cDKE

Reply:
LOL I realized I'd neglected to undo my masterpiece